### PR TITLE
Adds Pingdom Real User Monitoring (RUM) tag

### PR DESCRIPTION
**Pull request by sgiehl**
See https://help.pingdom.com/hc/en-us/articles/203611002-How-to-set-up-Real-User-Monitoring-RUM-

Was not able to test it properly as I don't have a Pingdom Professional or Enterprise account.

### CODE REVIEW

**Comment by Findus23** [comment]
Looking good.

The SVG could be replaced with 
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 82.53 66.43" height="66.43" width="82.53">
  <path d="M39.29 31.13l.7-.2c5.3-1.59 10.5-3.5 15.56-5.7a57.45 57.45 0 0 0 14.3-8.16 31.1 31.1 0 0 0 10.03-14.2c.42-.91.72-1.88.91-2.87-6.95 9.51-34.53 10.5-34.53 10.5l6.51-5.97A90.8 90.8 0 0 0 0 21.67a49.67 49.67 0 0 1 27.99 12.52c3.73-1.16 7.55-1.99 11.3-3.03zm43.24-.95a151.76 151.76 0 0 0-51.88 6.62 50.03 50.03 0 0 1 8.72 12.45c13.9-7.74 33.16-17.63 43.16-19.03zM41.13 53.2c1.64 4.25 2.7 8.7 3.13 13.23L68.5 41.52zm12.71 12.23v-.27c0-.1-.03-.2-.1-.3a.3.3 0 0 0-.24-.13h.21c.13-.09.2-.24.2-.4a.48.48 0 0 0-.3-.5 1.35 1.35 0 0 0-.5 0h-.69v1.88h.37v-.68h.26a.73.73 0 0 1 .35 0c.11.1.17.24.15.38v.28h.36l-.07-.26zm-1.07-1.46H53a.84.84 0 0 1 .38 0c.07.05.11.14.1.23a.25.25 0 0 1-.16.27h-.55v-.63z" fill="#282828" fill-rule="evenodd"/>
  <path d="M54.29 63.43a1.65 1.65 0 0 0-2.41 0c-.33.31-.5.75-.5 1.2 0 .46.17.9.5 1.21a1.65 1.65 0 0 0 2.41 0 1.65 1.65 0 0 0 0-2.41m-1.2 2.6c-.38.01-.74-.14-1-.4a1.42 1.42 0 0 1 0-1.99 1.39 1.39 0 0 1 1.99 0c.53.55.53 1.43 0 1.98-.26.27-.62.42-1 .42" fill="#282828" fill-rule="evenodd"/>
</svg>
```

**Comment by sgiehl** [comment]
I used the official logo provided by pingdom here https://www.pingdom.com/company/brand-assets/. But feel free to replace it

File: Template/Tag/PingdomRUMTag.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+use Piwik\Validators\NotEmpty;
+
+class PingdomRUMTag extends BaseTag
+{
+    public function getName() {
+        return "Pingdom Real User Monitoring (RUM)";
+    }
+
+    public function getCategory() {
+        return self::CATEGORY_OTHERS;
```
Comment (tsteur) [inline Template/Tag/PingdomRUMTag.php]: Would category be more analytics? or developers?

**Comment by tsteur** [comment]
I haven't tested it but looks good. Would maybe tweak the category.

File: Template/Tag/PingdomRUMTag.php (hunk 1 above)
Comment (tsteur) [reply Template/Tag/PingdomRUMTag.php]: Looks to me like Analytics, will move it there 👍

**Pull request merged**